pkg/parser/libfuzzer/stacktrace/stacktrace.go

```diff
@@ -106,7 +115,13 @@ func (p *parser) parseSourceLocation(logs []string) ([]*StackFrame, error) {
 }
 
 func (p *parser) stackFrameFromLine(line string) (*StackFrame, error) {
-	matches, found := regexutil.FindNamedGroupsMatch(framePattern, line)
+	var matches map[string]string
+	var found bool
+	if p.SupportJazzer {
+		matches, found = regexutil.FindNamedGroupsMatch(framePatternJava, line)
+	} else {
+		matches, found = regexutil.FindNamedGroupsMatch(framePattern, line)
+	}
```

Comment:
It looks like you accidentally removed my fixup commit instead of squashing it. Please squash this change into the `refactor: Parse stack trace in libFuzzer parser` commit.

```suggestion
	matches, found := regexutil.FindNamedGroupsMatch(framePattern, line)
        if !found && p.SupportJazzer {
                matches, found = regexutil.FindNamedGroupsMatch(framePatternJava, line)
                if !found {
                        return nil, nil
                }
        }
```

Reply:
I'm sorry, I was so sure I squashed it in, will fix it right away!